scripts/installscripts/install-jukebox.sh

```diff
@@ -1023,7 +1023,7 @@ install_main() {
     sudo systemctl disable phoniebox-rotary-encoder
     sudo systemctl disable phoniebox-gpio-buttons.service
     sudo rm "${systemd_dir}"/rfid-reader.service
-    sudo rm "${systemd_dir}"/startup-sound.service
+    sudo rm "${systemd_dir}"/phoniebox-startup-sound.service
```

Comment:
Shouldn’t this be phoniebox-startup-scripts?

Reply:
No, this is the removal of the not anymore existing daemons, during an update.
I fixed the name of the old one.